train_supervised.py

```diff
@@ -85,28 +86,27 @@
         if i % 5 == 0:
             tqdm.write(f"loss: {loss.item()}")
 
-
     if epoch % eval_every == 0:
-        #if i % 10 == 0:
-            #total = 0
+        # if i % 10 == 0:
+        # total = 0
         corrects = []
         losses = []
         with torch.no_grad():
             model.eval()
-            #check accuracy on validation set
-            for data in testloader:
-                images, labels = data[0].to(device), data[1].to(device)
-                logits = model(images)
-                loss = nn.CrossEntropyLoss(reduction='none')(logits, labels).cpu().numpy()
+            # check accuracy on validation set
+            for test_inputs, test_labels in testloader:
+                test_inputs, test_labels = test_inputs.to(device), test_labels.to(device)
+                logits = model(test_inputs)
+                loss = nn.CrossEntropyLoss(reduction='none')(logits, test_labels).cpu().numpy()
```

Comment:
Always best to use new variable names (i.e. `test_labels` instead of reusing `labels`) instead of overloading variable names.

Sometimes it's handy, like when performing a series of related operations (for example, look at lines 70 through 85 in `resnet_official` - that's a fine use of overloading).  But in this case, I'd worry about accidentally using train labels when test labels were intended, or vice versa.